src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,88 @@
+package duke;
+
+import duke.parser.Parser;
+import duke.task.TaskManager;
+import storage.Storage;
+
+import java.io.IOException;
+import java.util.Locale;
+import java.util.Scanner;
+
+public class Duke {
+    public static final String FILE_PATH = "duke.txt";
+    public static final String DIVIDER = "-----------------------------------------";
+
+    public static void main(String[] args) {
+        printWelcomeMessage();
+        TaskManager taskManager = loadSavedFile();
```

Comment:
You wish to initialize a taskManager but are loading a saved file instead. Consider a replacing `loadSavedFile()` with a more [meaningful name](https://nus-cs2113-ay2122s1.github.io/website/se-book-adapted/chapters/codeQuality.html#use-nouns-for-things-and-verbs-for-actions`. ).

Hint: What inputs are required to construct a taskManager?